tensorboard/plugins/graph/keras_util.py

```diff
@@ -41,18 +40,41 @@
 Please refer to https://github.com/tensorflow/tfjs-layers/blob/master/src/keras_format/model_serialization.ts
 for more complete definition.
 """
+import collections
+import six
+
 from tensorboard.compat.proto.graph_pb2 import GraphDef
 from tensorboard.compat.tensorflow_stub import dtypes
 
 
+# Please refer to below link for more information.
+# https://github.com/tensorflow/tensorflow/blob/1020739e1724346ef0088186f0253defaad3edb6/tensorflow/python/keras/engine/network.py#L1079-L1090
+InboundNodes = collections.namedtuple(
```

Comment:
Is `InboundNodesEntry` or `InboundEdge` or similar a more appropriate
name here? Nodes themselves are never really “inbound”, right?